<?xml version="1.0"?>
<launch>
  <!-- configs -->
  <param name="debug_mode" type="bool" value="true" />
  <param name="node_config" type="str" value="$(find awesomo_ros)/configs/default/apriltag_node.yaml" />
  <param name="apriltag_config" type="str" value="$(find awesomo_configs)/configs/apriltag/config.yaml" />

  <!-- publish topics -->
  <param name="apriltag_pose_topic" type="str" value="awesomo/apriltag/pose" />

  <!-- subscribe topics -->
  <param name="shutdown" type="str" value="awesomo/apriltag/shutdown" />
  <param name="camera_pose_topic" type="str" value="awesomo/camera/image" />

  <!-- node -->
  <node pkg="awesomo_ros" name="awesomo_apriltag" type="apriltag_node" output="screen" />
</launch>
